vars/RedHat-9.yml

```diff
@@ -31,4 +31,5 @@ __cockpit_packages_exclude:
   - cockpit-docker   # omit in favor of podman
   - cockpit-ostree   # omit only needed for CoreOS
   - cockpit-tests    # omit only needed for testing
+  - cockpit-pcp      # not present any more in current composes, but in older ones, and conflicts with current bridge
```

Comment:
What will happen when someone tries to use this role to manage a 9.2 or 9.4 system?

Reply:
I think the official answer is "use rhel-system-roles from 9.2 or 9.4", as only those are supported, right? Upstream/on main we don't test 9.2 or 9.4, just CentOS (and thus approximately "latest RHEL"), so they will always be potentially broken.

For this particular case, cockpit-pcp will then not be installed any more. This is also somewhat good, as this pulls in pcp which is eats significant resources on a system (and you may not even be aware).

Reply:
Done in PR #206